aiohttp/web_jobs.py

```diff
@@ -0,0 +1,223 @@
+import asyncio
+import sys
+import traceback
+from collections import deque
+from collections.abc import Container
+
+import async_timeout
+
+from .helpers import ensure_future
+
+
+__all__ = ['JobRunner']
```

Comment:
Technically better to remove `__all__` at all -- both `Job` and `JobRunner` are not exposed to `aiohttp.web` namespace.